cluster/examples/kubernetes/ceph/import-external-cluster.sh

```diff
@@ -5,6 +5,11 @@ set -e
 # VARIABLES #
 #############
 MON_SECRET_NAME=rook-ceph-mon
+MON_CHECKER_SECRET_NAME=rook-ceph-mon-checker
```

Comment:
For the import can we just call this the `OPERATOR_CREDS=rook-ceph-operator-creds`? If we keep it generic it will make sense in case they use a non-default username and also in the future when we make it a more general solution.